### ISSUE TITLE
[Chain Listings] Celestia Support - Archival Nodes

### ISSUE THREAD

**Issue by RossiNYC** [pokt-network/poktroll#281]
## Objective

Get Celestia Archival (Full Storage Data Availability Node) listed as a supported ServiceID 
Explore possible grant opportunities to help bootstrap node runners/gateways

## Origin Document

[Celestia Node Tutorial Docs](https://docs.celestia.org/developers/node-tutorial)

![Image](https://github.com/pokt-network/poktroll/assets/95501635/55d54857-8835-4c25-bf6a-2045f1038750)

## Goals

1. Support Celestia Archival  (Full Storage Data Availability Node) 
2. Grant

## Deliverables

- [x] PNF Gov Tx to add ServiceID 
- [x] Grant $

## Non-goals / Non-deliverables

- Setting up an actual Celestia Archive node

**Creator**: [@RossiNYC]
**Co-Owners**: [@Olshansk]

**Comment by RossiNYC** [comment]
## Objective

Get Celestia Archival (Full Storage Data Availability Node) listed as a supported ServiceID 
Explore possible grant opportunities to help bootstrap node runners/gateways

## Origin Document

[Celestia Node Tutorial Docs](https://docs.celestia.org/developers/node-tutorial)



![Image](https://github.com/pokt-network/poktroll/assets/95501635/55d54857-8835-4c25-bf6a-2045f1038750)



## Goals

1. Support Celestia Archival  (Full Storage Data Availability Node) 
2. Grant

## Deliverables

- [ ] PNF Gov Tx to add ServiceID 
- [ ] Grant $

## Non-goals / Non-deliverables

- Setting up an actual Celestia Archive node

**Creator**: [@RossiNYC]
**Co-Owners**: [@Olshansk]

**Comment by Olshansk** [comment]
Any chance we can do this earlier?

![Screenshot 2023-12-15 at 10 16 41 AM](https://github.com/pokt-network/poktroll/assets/1892194/6e54d49f-9430-4ff1-a570-e462898b4110)

**Comment by RossiNYC** [comment]
@Olshansk yes, will move it up to Iteration 8

**Comment by RossiNYC** [comment]
@misskitty42 and @icculp reached out, shared the link to this ticket with Zak from PNF so he can forward to others who contacted him and drop a discord update to comment here if there is interest in running a node.

**Comment by misskitty42** [comment]
Yeah, I'm interested in running a node.  I'm sourcing hardware right now.  When would you like it to be live?

**Comment by Olshansk** [comment]
> Yeah, I'm interested in running a node. I'm sourcing hardware right now. When would you like it to be live?

No specific time/date (figuring out logistics), but could be as soon as within a few weeks!

**Comment by Olshansk** [comment]
> Yeah, I'm interested in running a node. I'm sourcing hardware right now. When would you like it to be live?

No specific time/date (figuring out logistics), but could be as soon as within a few weeks!

@misskitty42 Do you have any details on what the hardware requirements are looking like?

**Comment by misskitty42** [comment]
> > Yeah, I'm interested in running a node. I'm sourcing hardware right now. When would you like it to be live?
> 
> No specific time/date (figuring out logistics), but could be as soon as within a few weeks!
> 
> @misskitty42 Do you have any details on what the hardware requirements are looking like?

Per the Celestia docs, full storage node requirements are: 
Memory: 4 GB RAM (minimum)
CPU: 6 cores
Disk: 10 TB SSD Storage
Bandwidth: 1 Gbps for Download/1 Gbps for Upload

I would assume disk space ought to be at least 16TB (if not 20) to allow for db expansion.  I wish they would state specifically what kind of SSD, but I'll play it safe and assume they are referring to nVME.

**Comment by misskitty42** [comment]
Update:  hardware procured, DA node is now syncing.

**Comment by icculp** [comment]
I have a consensus archival and da bridge sycned as of last weekend, I shot a message to someone from PNF about it but I think it slipped through the cracks

**Comment by Olshansk** [comment]
Thanks for the updates @misskitty42 and @icculp. We're pushing things through on our end and should get this going soon!

**Comment by Olshansk** [comment]
@moatus We can close this out when TestNet nodes are supported.

**Comment by moatus** [comment]
Node runners expect to finish syncing this week for support of MainNet and TestNet in US, EU, and AP. Still to do:

- [ ] Whitelist Celestia TestNet
- [ ] Grove launch Celestia TestNet support

**Comment by Olshansk** [comment]
@fredteumer FYI ☝️

**Comment by Olshansk** [comment]
@moatus Let us know once this is finished off.

**Comment by moatus** [comment]
Allowlisting scheduled for tomorrow. This is now complete.
